services/tasks.js

```diff
@@ -2,33 +2,8 @@ const firestore = require("../utils/firestore");
 const tasksModel = firestore.collection("tasks");
 const { chunks } = require("../utils/array");
 const { DOCUMENT_WRITE_SIZE: FIRESTORE_BATCH_OPERATIONS_LIMIT } = require("../constants/constants");
-
-const updateTaskStatusToDone = async (tasksData) => {
```

Comment:
Why are you removing this?

Reply:
It was causing circular dependency issues since it was being imported in models from services, and since it's directly interacting with the database, it should be placed in the models.